pkgs/by-name/un/undbx/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, autoreconfHook
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "undbx";
+  version = "0.22-unstable-2019-02-11";
+
+  src = fetchFromGitHub {
+    owner = "ZungBang";
+    repo = "undbx";
+    rev = "5e31c757e137a6409115cac0623d61d384019b7a";
+    hash = "sha256-leregcv3dv/D3WvFkYyjQePdKi4BgE0aj5PY6JiSKl8=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  patches = [
+    # change Makefile.am to allow the GCC hardening.
+    (fetchpatch {
+      url = "https://sources.debian.org/data/main/u/undbx/0.21-4/debian/patches/010_fix-makefile.patch";
+      hash = "sha256-tVrluH96R/LGhCnExaGUALm0BsPQtEuXduItnTHGL38=";
+    })
+  ];
+
```

```suggestion
  postPatch = ''
    substituteInPlace Makefile.am \
      --replace-fail "-Werror" "" \
      --replace-fail "bin_SCRIPTS" "#bin_SCRIPTS"
  '';
```

i am not sure if sources.debian.org urls are stable
https://nixos.org/manual/nixpkgs/unstable/#fetchdebianpatch
suggests they are not.